nemo/collections/common/metrics/sacrebleu.py

```diff
@@ -0,0 +1,1928 @@
+#!/usr/bin/env python3
```

Comment:
can you please fix copy-right header:
1) remove #!/usr/bin/env ....  line
2) remove # ========= ... lines

Reply:
I have trouble with copy-right in file [`nemo/collections/nlp/metrics/sacrebleu.py`](https://github.com/PeganovAnton/NeMo/blob/mt/nemo/collections/nlp/metrics/sacrebleu.py). There is an Amazon string. What should I do with it?